### ISSUE TITLE
A11y_SQLAzureDataStudio_Extension_Install-UninstallExtension_ScreenReader: Heading content not reading as heading by screen reader.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9258]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #SQL Azure Data Studio; #DesktopApp; #Win32; #MAS1.3.1; #Benchmark; #A11y_SQLAzureDataStudio; #A11ySev3; #NVDA; #JAWS; #MAS2.4.6;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5e5f8423d3249670f55b1466b1980a5f03
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen Reader: NVDA(2019.3.1),JAWS
MAS reference: MAS1.3.1,MAS2.4.6

**Repro Steps:**
1. Launch Azure Data Studio.
2. From the left menu, click the 'Extensions' icon to open the Extension Manager.
3. Choose any extension from the library.
4. Click the green 'Install' button to install.
5. Launch NVDA/JAWS screen reader and navigate to the heading 'Admin pack for SQL Server'.
6. Check if all headings are announced by screen reader.

**Actual:**
Heading content is not announced as heading to screen reader users present at the top of the screen. "Admin pack for SQL Server".

**Issue observed with both the screen readers JAWS and NVDA.**

**Expected:**
Heading content should be announced as heading by screen reader.

**User Impact:**
Screen reader users might miss the heading content present on the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference:
MAS 1.3.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**MAS 2.4.6**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}

**Attachment link reference:**
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-atkulk_microsoft_com/ETFY1XtkAdBCrAkfNGt0XUkBkopuoTQMVXEYp3YeQnaoBw?e=daA81e

**Comment by alanrenmsft** [comment]
dupe of #9411

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
